### PR TITLE
Kevin/convo endpoint

### PR DESCRIPTION
**Pull request by kevoconnell**
adds conversation endpoint info

### CODE REVIEW

File: src/v2/spec.yaml [hunk 1]
```diff
@@ -72,6 +72,17 @@ components:
         The query param accepted by the API. Sent along with identifier param.
         url - Cast identifier is a url
         hash - Cast identifier is a hash
+    FilterSpam:
```
Comment (rishavmukherji) [inline src/v2/spec.yaml:75]: we dont expose spam filtering to developers yet, we only have it internally right now. lets take this out and we will expose it once we are ready

File: src/v2/spec.yaml [hunk 2]
```diff
@@ -1982,6 +2013,17 @@ paths:
           example: 0xa6a8736f18f383f1cc2d938576933e5ea7df01a1,0x7cac817861e5c3384753403fb6c0c556c204b1ce
           schema:
             type: string
+        - name: address_types
+          description: Customize which address types the request should search for. Default is all. Select multiple types by using a comma-separated list of valid types in the enum
```
Comment (rishavmukherji) [inline src/v2/spec.yaml:2017]: isn't this a different PR?

File: src/v2/spec.yaml [hunk 3]
```diff
@@ -2102,6 +2144,51 @@ paths:
...
+          example: url
+          schema:
+            $ref: "#/components/schemas/CastParamType"
+        - name: levels
+          in: query
+          required: false
+          description: "levels up and down to fetch conversations"
+          schema:
+            $ref: "#/components/schemas/Levels"
+        - name: filter_spam
```
Comment (rishavmukherji) [inline src/v2/spec.yaml:2177]: yeah no need to expose filter spam yet

File: src/v2/spec.yaml [hunk 4]
```diff
@@ -2102,6 +2144,51 @@ paths:
...
+        - name: type
+          required: true
+          in: query
+          example: url
+          schema:
+            $ref: "#/components/schemas/CastParamType"
+        - name: levels
+          in: query
+          required: false
+          description: "levels up and down to fetch conversations"
```
Comment (rishavmukherji) [inline src/v2/spec.yaml:2174]: we should add the min and max in the description so looking at it, developers know what the range is

**Review by rishavmukherji — CHANGES_REQUESTED** [review]

File: src/v2/spec.yaml (hunk 2 above)
Comment (MrKevinOConnell) [reply src/v2/spec.yaml:2010]: yeah I saw that - i think thats below, not sure why its showing, should have been up to date 🤔  @rishavmukherji 

https://github.com/neynarxyz/OAS/pull/51

File: src/v2/spec.yaml (hunk 4 above)
Comment (MrKevinOConnell) [reply src/v2/spec.yaml]: this should be in the Levels object @rishavmukherji

**Review by manan19 — APPROVED** [review]

**Pull request merged**